main.go

```diff
@@ -8,8 +8,13 @@ package main
 
 import (
 	"github.com/operate-first/opfcli/cmd"
+	"github.com/operate-first/opfcli/utils"
+	"github.com/spf13/cobra"
 )
 
 func main() {
-	cmd.Execute()
+	utils.ConfigureLogging()
+
+	cmd := cmd.NewCmdRoot()
```

Comment:
can you rename `cmd` here to something else so it's not colliding with the imported pkg?